addons/l10n_jp/data/account.fiscal.position.template.csv

```diff
@@ -1,4 +1,4 @@
-id,chart_template_id/id,name
-fiscal_position_tax_inclusive_template,l10n_jp1,内税
-fiscal_position_tax_exclusive_template,l10n_jp1,外税
-fiscal_position_tax_exempt_template,l10n_jp1,海外取引先
+id,chart_template_id/id,name,sequence,auto_apply,country_id:id
```

Comment:
With the inclusive and exclusive taxes, we can not know which one is preferred.  So, I would leave it out, unless @yostashiro maybe you know if these 2 fiscal positions make sense or which one takes precedence to apply automatically.